MaterialLibrary/src/main/java/io/doist/material/widget/FloatingActionButton.java

```diff
@@ -1,51 +1,39 @@
 package io.doist.material.widget;
 
 import android.annotation.SuppressLint;
-import android.annotation.TargetApi;
 import android.content.Context;
 import android.content.res.ColorStateList;
 import android.content.res.TypedArray;
-import android.graphics.Canvas;
-import android.graphics.Color;
-import android.graphics.Paint;
-import android.graphics.Point;
-import android.graphics.RadialGradient;
-import android.graphics.Shader;
 import android.graphics.drawable.Drawable;
 import android.graphics.drawable.GradientDrawable;
-import android.graphics.drawable.InsetDrawable;
 import android.graphics.drawable.RippleDrawable;
 import android.os.Build;
-import android.support.annotation.NonNull;
 import android.util.AttributeSet;
-import android.util.DisplayMetrics;
-import android.util.TypedValue;
-import android.view.Display;
 import android.view.View;
 import android.view.ViewGroup;
-import android.view.ViewTreeObserver;
-import android.view.WindowManager;
 import android.widget.ImageButton;
 
 import io.doist.material.R;
 import io.doist.material.color.ColorPalette;
 import io.doist.material.drawable.RippleMaterialDrawable;
 import io.doist.material.drawable.TintDrawable;
+import io.doist.material.elevation.ElevationDelegate;
 
 public class FloatingActionButton extends ImageButton {
-    private static final int DEFAULT_ELEVATION_DP = 6;
+    public static final String LOG_TAG = FloatingActionButton.class.getSimpleName();
 
-    // Used to create a shadow to fake elevation in pre-L androids.
-    private ElevationManager mElevationManager;
+    private static final int DEFAULT_ELEVATION_DP = 8;
```

Comment:
So… appcompat’s fab elevation is still `8dip` :expressionless: https://cloudup.com/crTXJ3surY6

Will change manually yo `6dip`. When we use a `dimen` for this, I think it should be appcompat's. Hopefully this will be fixed soon.